src/sandbox.rkt

```diff
@@ -402,9 +402,10 @@
      :herbie-error-output
      ([,(*num-points*) ,(table-row-result-est row)]
       [,(*reeval-pts*) ,(table-row-result row)])
-     ,@(if (table-row-target row)
-           `(:herbie-error-target ([,(*reeval-pts*) ,(table-row-target row)]))
-           '())
+     ,@(append
+        (for/list ([rec (in-list (table-row-target row))])
```

Comment:
I see. So before the list wasn't getting unrolled?

Reply:
Yeah, we were just dumping `:herbie-error-target ([8000 ()])` for every benchmark, even if there were no targets.